From eeb772d89e6b159b8152cf08afb7503addb7fbfa Mon Sep 17 00:00:00 2001
From: James McCoy <jamessan@debian.org>
Date: Sat, 14 May 2022 16:57:22 -0400
Subject: openssl-3.x

No DEP3 Subject or Description header found

Gbp-Pq: openssl-3.x.
---
 buckets/ssl_buckets.c | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/buckets/ssl_buckets.c b/buckets/ssl_buckets.c
index b01e535..5a45868 100644
--- a/buckets/ssl_buckets.c
+++ b/buckets/ssl_buckets.c
@@ -1325,9 +1325,14 @@ static int ssl_need_client_cert(SSL *ssl, X509 **cert, EVP_PKEY **pkey)
                 return 0;
             }
             else {
+#ifdef ERR_GET_FUNC
                 printf("OpenSSL cert error: %d %d %d\n", ERR_GET_LIB(err),
                        ERR_GET_FUNC(err),
                        ERR_GET_REASON(err));
+#else
+                printf("OpenSSL cert error: %d %d\n", ERR_GET_LIB(err),
+                       ERR_GET_REASON(err));
+#endif
                 PKCS12_free(p12);
                 bio_meth_free(biom);
             }
-- 
cgit v1.1

